### PR TITLE
Updated Launching-new-feature wiki page.

### PR DESCRIPTION
**Pull request by lkbhitesh07**
As we have moved the features tab from the admin page to the release=coordinator page we would require to update the wiki page.

### CODE REVIEW

**Comment by lkbhitesh07** [comment]
Hey @seanlip, done with the changes.
Thanks

File: Launching-new-features.md [hunk 1]
```diff
@@ -23,17 +23,17 @@ Say you are working on a large scale user-facing feature that will take 1 or mor
 
 2. The first PR above must be merged before any of the following PRs are merged. This is to ensure that the feature flag is available in the codebase for it to be used in the following PRs.
 
-3. When developing, make sure that e2e tests are present for your feature. If the feature is gated behind a feature flag, you may need to use the 'enableFeatureForProd'/'enableFeatureForDev' utility functions for the admin page to first enable the required flag, and then proceed to perform the testing. For unit tests, you should include tests for both the `flag=True` and `flag=False` cases.
+3. When developing, make sure that e2e tests are present for your feature. If the feature is gated behind a feature flag, you may need to use the 'enableFeature' utility functions for the release-coordinator page to first enable the required flag, and then proceed to perform the testing. For unit tests, you should include tests for both the `flag=True` and `flag=False` cases.
```
Comment (seanlip) [inline Launching-new-features.md]: Suggest linking to it: https://github.com/oppia/oppia/blob/c48ff1510a680666bfe891d7b2f68130d21e4dcf/core/tests/webdriverio_utils/ReleaseCoordinatorPage.js#L126

File: Launching-new-features.md [hunk 2]
```diff
@@ -23,17 +23,17 @@ Say you are working on a large scale user-facing feature that will take 1 or mor
...
 
-7. Once you receive a go-ahead from the feature testers, you must merge another PR. This PR should do only one thing, i.e. move the feature flag to the PROD stage, allowing it to be enabled/disabled in production (by the admin(s)). **NOTE: When opening this PR, include a link to the testing doc or other proof that the feature has been approved for release.** While this PR is open, confirm with the release coordinators that the new CUJs for this feature have been added to the overall CUJs used for testing releases in general.
+7. Once you receive a go-ahead from the feature testers, you must merge another PR. This PR should do only one thing, i.e. move the feature flag to the PROD stage, allowing it to be enabled/disabled in production (by the release-coordinator(s)). **NOTE: When opening this PR, include a link to the testing doc or other proof that the feature has been approved for release.** While this PR is open, confirm with the release coordinators that the new CUJs for this feature have been added to the overall CUJs used for testing releases in general.
```
Comment (seanlip) [inline Launching-new-features.md]: No hyphen between "release" and "coordinators".

File: Launching-new-features.md [hunk 3]
```diff
@@ -23,17 +23,17 @@ Say you are working on a large scale user-facing feature that will take 1 or mor
...
 
-8. Once this PR is merged, send a ["job run request"](https://forms.gle/rUJaHJSpRGemtGDp6) to the release coordinators to turn on the feature in production by adding a rule in the `/admin` page.
+8. Once this PR is merged, send a ["job run request"](https://forms.gle/rUJaHJSpRGemtGDp6) to the release coordinators to turn on the feature in production by adding a rule in the `/release-coordinator` page.
```
Comment (seanlip) [inline Launching-new-features.md:36]: Why still "adding a rule"?

File: Launching-new-features.md [hunk 4]
```diff
@@ -311,21 +311,21 @@ if (this.featureService.status.NewFeature.isEnabled) {
 
 ## Changing Value of Feature Flags
 
-Feature flags are defaulted to `false/disabled`. To change their values, you can login as the administrator, navigate to the admin page, then to the feature tab.
+Feature flags are defaulted to `false/disabled`. To change their values, you can login as the administrator, provide yourself the role of release-coordinator from the 'Roles' tab present on the Admin page, navigate to the release-coordinator page, then to the feature tab.
```
Comment (seanlip) [inline Launching-new-features.md]: The first release coordinator should be "Release Coordinator" (same as Admin).

The second one should be /release-coordinator (with a leading slash) and maybe you can put that in backticks: `/release-coordinator`.

File: Launching-new-features.md [hunk 5]
```diff
@@ -311,21 +311,21 @@ if (this.featureService.status.NewFeature.isEnabled) {
...
 
 In the feature tab, where you will see the feature flag you added, you can change the settings (see the [Setting of Feature Flags](#settings-of-feature-flags) section for detail) of the feature flags.
 
-Note: since only users with admin permission can edit the settings of feature flags, you can only enable your features on your local dev instance of Oppia, while in production environment, only administrators can enable/disable features.
+Note: since only users with release-coordinator permission can edit the settings of feature flags, you can only enable your features on your local dev instance of Oppia, while in production environment, only release-coordinators can enable/disable features.
```
Comment (seanlip) [inline Launching-new-features.md]: The second of these should have no hyphen.

Let's also break this up a bit: ...of Oppia. However, on the live site, only release coordinators ...

**Review by seanlip — CHANGES_REQUESTED** [review]
Thanks @lkbhitesh07, left some notes. PTAL.

File: Launching-new-features.md (hunk 1 above)
Comment (lkbhitesh07) [reply Launching-new-features.md]: Done

File: Launching-new-features.md (hunk 2 above)
Comment (lkbhitesh07) [reply Launching-new-features.md]: Done

File: Launching-new-features.md (hunk 3 above)
Comment (lkbhitesh07) [reply Launching-new-features.md:36]: The UI for features is not yet updated and will be done as part of M2 of my project, So in order to enable the feature we still have to add a rule.
Thanks

File: Launching-new-features.md (hunk 4 above)
Comment (lkbhitesh07) [reply Launching-new-features.md]: Done

File: Launching-new-features.md (hunk 5 above)
Comment (lkbhitesh07) [reply Launching-new-features.md]: Done

**Review by seanlip — APPROVED** [review]
Looks great. Thanks!

**Pull request merged**